test/EFTools/UnitTests/EntityDesignerVersioningFacade/ReverseEngineerDb/StoreModelBuilderTests.cs

```diff
@@ -785,37 +785,37 @@ public void CreateProperties_creates_properties_for_valid_rows_and_exclude_prope
             Assert.Equal("GeographyKey", invalidKeyTypeColumns[0]);
         }
 
-        [Fact]
+        [Fact(Skip = "Different API Visiblity between official dll and locally built")]
         public void Build_creates_EdmModel_containing_converted_objects()
         {
-            var tableDetails = new[] { CreateRow(null, "dbo", "table", "Id", 0, false, "int", isIdentiy: true) };
-            var viewDetails = new[] { CreateRow(null, "dbo", "view", "Id", 0, false, "int", isIdentiy: true) };
-            var relationshipDetails = new[]
-                { CreateRelationshipDetailsRow("id", "name", 0, false, null, "dbo", "table", "Id", null, "dbo", "view", "Id") };
-            var functionDetails = new[] { CreateFunctionDetailsRow(functionName: "function", isTvf: true) };
-            var tvfReturnTypeDetails = new[] { CreateRow(null, null, "function", "age", 0, false, "int") };
-
-            var storeModel =
-                CreateStoreModelBuilder(
-                    namespaceName: "my.Model",
-                    targetEntityFrameworkVersion: EntityFrameworkVersion.Version3)
-                    .Build(new StoreSchemaDetails(tableDetails, viewDetails, relationshipDetails, functionDetails, tvfReturnTypeDetails));
-
-            Assert.NotNull(storeModel);
-            Assert.Equal(3.0, storeModel.SchemaVersion);
-            Assert.Equal("System.Data.SqlClient", storeModel.ProviderInfo.ProviderInvariantName);
-            Assert.Equal("2008", storeModel.ProviderInfo.ProviderManifestToken);
-            Assert.Equal("myModelContainer", storeModel.Containers.Single().Name);
-            Assert.Equal(2, storeModel.Containers.Single().EntitySets.Count);
-            Assert.Equal(1, storeModel.Containers.Single().AssociationSets.Count);
-            Assert.Equal(2, storeModel.EntityTypes.Count());
-            Assert.Equal(1, storeModel.AssociationTypes.Count());
-            Assert.Equal(1, storeModel.Functions.Count());
-
-            var returnParameter = storeModel.Functions.Single().ReturnParameter;
-            Assert.IsType<CollectionType>(returnParameter.TypeUsage.EdmType);
-            Assert.IsType<RowType>(
-                ((CollectionType)returnParameter.TypeUsage.EdmType).TypeUsage.EdmType);
+            //var tableDetails = new[] { CreateRow(null, "dbo", "table", "Id", 0, false, "int", isIdentiy: true) };
```

Comment:
is there a risk with the tests that are being skipped and commented out in this PR?

Reply:
Here's the plan:
The 6.2.0 of EntityFramework.* packages is quite old; updating for the modern VS makes the whole system healthier.  With each change we make for product reasons, we'll ensure adjoining test changes.  That is, if no test had to be altered because of product changes, author a new test OR find a set of tests to unskip by coming up with a way to get the right mock behavior with a type that lacks a parameterless constructor (or other example).  We will see a localized drop in unittest coverage from a theoretical perspective, but previous to this the tests wouldn't even run in VS, so on the whole it feels like coverage just jumped up and we'll continue to improve it over time.